tests/unit/src/unit/TestRemoting.hx

```diff
@@ -12,9 +12,9 @@ class TestRemoting extends Test {
 	static var _ : Dynamic = init();
 	static var ecnx : haxe.remoting.ExternalConnection;
 	static var ecnx2 : haxe.remoting.ExternalConnection;
-	static var ecnx3 : haxe.remoting.ExternalConnection;
+	// static var ecnx3 : haxe.remoting.ExternalConnection;
```

Comment:
As far as I understood, they are somehow using flash8 iframe in unit.html or something and we removed that, but I'm not sure what to do with these tests.